index.js

```diff
@@ -264,7 +266,14 @@ class HtmlWebpackInlineSVGPlugin {
             // process the imageNodes
 
             this.updateHTML(html, inlineImages)
-                .then((html) => resolve(html))
+                .then((html) => {
+                    if (!this.allowFromUrl) {
+                        resolve(html)
+                    }
+
+                    const newHtml = html.replace(/failed /gi, '')
+                    resolve(newHtml)
```

Comment:
@AntonioRedondo Each image retrieval that failed got marked with the inline attribute `failed` so that it gets restored at the end

Reply:
This entire `.then((html) => { //** */ }` block can be replaced with:
```js
.then((html) => resolve(this.allowFromUrl ? html.replace(/failed /gi, '') : html))
```

Reply:
On the other hand I think it's better if the `failed` attribute is left on the element and the code remains as it was before. It won't cause any problem to the browser render and it will tell the developer that there is an issue with the image.